packages/gatsby/cache-dir/static-entry.js

```diff
@@ -62,51 +51,16 @@ const getPageDataUrl = pagePath => {
   return `${__PATH_PREFIX__}/${pageDataPath}`
 }
 
-const getStaticQueryUrl = hash =>
-  `${__PATH_PREFIX__}/page-data/sq/d/${hash}.json`
-
-const getPageData = pagePath => {
-  const pageDataPath = getPageDataPath(pagePath)
-  const absolutePageDataPath = join(process.cwd(), `public`, pageDataPath)
-  const pageDataRaw = fs.readFileSync(absolutePageDataPath)
-
-  try {
-    return JSON.parse(pageDataRaw.toString())
-  } catch (err) {
-    return null
-  }
-}
-
-const appDataPath = join(`page-data`, `app-data.json`)
-
-const getAppDataUrl = memoize(() => {
-  let appData
+const getStaticQueryPath = hash => join(`page-data`, `sq`, `d`, `${hash}.json`)
 
-  try {
-    const absoluteAppDataPath = join(process.cwd(), `public`, appDataPath)
-    const appDataRaw = fs.readFileSync(absoluteAppDataPath)
-    appData = JSON.parse(appDataRaw.toString())
-
-    if (!appData) {
-      return null
-    }
-  } catch (err) {
-    return null
-  }
+const getStaticQueryUrl = hash =>
+  `${__PATH_PREFIX__}/${getStaticQueryPath(hash)}`
 
-  return `${__PATH_PREFIX__}/${appDataPath}`
-})
+const getAppDataUrl = () =>
+  `${__PATH_PREFIX__}/${join(`page-data`, `app-data.json`)}`
```

Comment:
This also probably produces a bad URL on Windows?

Reply:
switched to `path.posix` in https://github.com/gatsbyjs/gatsby/pull/29396/commits/8212f33ce5d9298e6cb18067823d01652ec03bf2